lib/index.tsx

```diff
@@ -15,6 +15,8 @@ interface AppstoreContextType {
     input: MakeApiRequestParams,
     skip: boolean
   ) => { refetch: any; data: any };
+  getAppUserDatastore: () => Promise<object>;
+  setAppUserDatastore: (data: object) => Promise<void>;
```

Comment:
do we not have a type for this "data" object?

Reply:
Oh it should be string type, app developer stringify userDateStore and submit it.

Reply:
this should actually just be a string, we want them to send data back as json